code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -484,14 +484,19 @@
 
 	. = 0
 
+///Determines the rate at which Plasma Fixation heals based on the amount of plasma in the air
+#define HEALING_PER_MOL 1.1
+
```

Comment:
My final nitpick:
Move the define to above `/datum/symptom/heal/plasma`'s declaration, and the undef to below `/datum/symptom/heal/plasma/Heal`'s implementation (above rad healing) 

Just keeps the defines nice and orderly instead of dropping them in the middle of the code

Reply:
If we're talking about segmenting off sections of code, would it not be better to place definitions either within the `/datum` or have all definitions cloistered together at the top of the page so that we're not skipping through looking for individual items, rather than outside of a datum, because IMO it's falling into the section of the file belonging to water healing.

Kind of like how in bug report templates everything relevant to that section is under the title in bold.